node/src/types/error.rs

```diff
@@ -142,3 +142,15 @@ pub(crate) enum BlockWithMetadataValidationError {
     #[error(transparent)]
     BlockHeaderWithMetadataValidationError(#[from] BlockHeaderWithMetadataValidationError),
 }
+
+#[derive(Error, Debug, PartialEq, Eq)]
+pub(crate) enum BlockHeadersBatchValidationError {
+    #[error("Batch has incorrect lenght. Expect: {expected}, got: {got}")]
```

Comment:
```suggestion
    #[error("Batch has incorrect length. Expected: {expected}, got: {got}")]
```

Reply:
Seems like I got rid of this enum along the way. Resolving the comment.